build.sh

```diff
@@ -121,6 +124,13 @@ if [ "${B_CORE}" = "y" ]; then
 	"./phoenix-rtos-build/build-core-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.sh"
 fi
 
+#
+# Build test part
+#
+if [ "${B_TEST}" = "y" ]; then
+	b_build_test
```

Comment:
good idea with putting custom command here, I assume we will have custom test on per-project basis.